### DISCUSSION TITLE
Python也可以寫遊戲!!--pygame

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#836] [General]
學寫程式最容易產生興趣的方式，就是寫一個可以玩的遊戲。
程式寫起來，相當直覺
基本上只要會用class和function就可以寫很多東西了~

簡單介紹的文章：
https://hackmd.io/@andy010629/r103KC6Iv
https://blog.techbridge.cc/2019/10/19/how-to-build-up-game-with-pygame-tutorial/

**Comment by dinosaur1009** [comment]
感覺這個的確是新手可以入門的部分！

**Comment by Kotori0501** [comment]
謝謝分享
之前有打過貪吃蛇的程式，個人覺得還不錯。
但如果還可以學到更多相關遊戲的程式碼，我個人或覺得很開心，很有趣。

**Comment by HuangHsinTzu** [comment]
感謝分享🙏
這樣當家人問我程式學得怎麼樣的時候
就可以拿自己設計的遊戲給他們玩了！
他們一定會覺得很酷哈哈哈

**Comment by dabbie109303501** [comment]
寫遊戲的確可以讓人有繼續學程式的動力
而且用python處理需要圖片的東西很方便

**Comment by ching020721** [comment]
寫遊戲真的能激起動力繼續學習，遇到bug也比較有學習欲，會希望能夠解決問題只為了看到最終遊戲成果

**Comment by williamyen0623** [comment]
我自己是用Scratch堆疊程式方塊創作遊戲啟蒙的，做遊戲真的是很棒的學習方式，前提是要對遊戲有興趣

**Comment by manipulateed** [comment]
相比有些人都是喜歡遊戲才來想著怎麼寫一個遊戲，這篇文的介紹就是很好一個入門的通道。

**Comment by ruthchen1020** [comment]
之前有用別的程式寫過遊戲，感覺python是個簡單易上手的遊戲製作平台。

**Comment by evanchou1204** [comment]
用自己會的程式語言做出自已喜歡的遊戲，很酷很有成就感，會讓人很有動力想繼續學習python。

**Comment by wangtyphoon** [comment]
我曾經那過pygame寫貪食蛇，感覺這個套件適合寫互動機制較為簡單的遊戲!

**Comment by CPCpyt** [comment]
我覺得內容廣泛且淺顯易懂，
可以帶給對遊戲製作的同學很棒的啟發和動力。

**Comment by StevenWanglolz** [comment]
還記得當初用scratch拉積木製作的第一款小遊戲，當初做完真的很有成就感! 不過用python寫的話，應該會寫到想要撞牆哈哈哈

**Comment by Doreen331** [comment]
最近一直在想寒假到底要更深入學python還是物件導向的語言，只能說各有好壞，原本對python的印象是以數據分析或是爬蟲為主，但沒想到還可以寫遊戲

**Comment by BlahFish1006** [comment]
寫出一個自己喜歡的遊戲真的可以激發出較大的動力，去面對一個個關卡和一個個bug，
完成後一定很有成就感，還可以給親朋好友玩XD

**Comment by Tuoresa** [comment]
感謝分享！第一次接觸程式也是利用scratch寫遊戲而走入程式領域。
能夠創造出專屬於自己，且互動性極高的遊戲回饋，
肯定能為寫程式的人帶來極大的成就感阿阿阿阿阿！
